docs/ruleset_functions/is_baseline_system_1(HVAC_id).md

```diff
@@ -0,0 +1,58 @@
+        - Check if fansystem is constant volume, if yes then carry on: `if is_fan_CV(B_RMR, hvac_b.id) == TRUE:`  
+            - Check if the hvac system serves a single zone, if yes carry on: `if serves_single_zone(B_RMR, hvac_b.id) == TRUE:`   
+                - Get and create an object for the terminal unit associated with the zone: `terminal_b = get_terminal_unit_for_SZ_baseline_HVAC(B_RMR,hvac_b.id)`  
+                - Check that the terminal unit object does not equal Null: `if terminal_b != Null:`  
+                    - Check that the data elements associated with the terminal unit align with system 1: `if is_terminal_heat_source_none(B_RMR,terminal_b.id) ==  TRUE AND is_terminal_cool_source_none(B_RMR,terminal_b.id) == TRUE And is_terminal_fan_null(B_RMR,terminal_b.id) == TRUE AND is_terminal_supply_ducted(B_RMR,terminal_b.id) == FALSE :`      
+                        - if coolingsystem is DX and the heating fluid loop serves a boiler and the terminal type is constant volume then SYS-1: `if is_cooling_type_DX(B_RMR, hvac_b.id) == TRUE AND is_fluid_loop_attached_to_boiler(B_RMR, hvac_b.id) == TRUE AND is_terminal_type_CAV(B_RMR, terminal_b.id) ==  TRUE: is_baseline_system_1 = "SYS-1"`
+                        - elif coolingsystem is DX and the fluid loop is purchased heating and the terminal type is constant volume then SYS-1b: `elif is_cooling_type_DX(B_RMR, hvac_b.id) == TRUE AND is_fluid_loop_purchased_heating(B_RMR, hvac_b.id) == TRUE AND is_terminal_type_CAV(B_RMR, terminal_b.id) ==  TRUE: is_baseline_system_1 = "SYS-1b"`
+                        - elif the cooling system is a fluid loop and terminal type is four pipe: `elif is_cooling_type_fluid_loop(B_RMR, hvac_b.id) == TRUE AND is_terminal_type_four_pipe(B_RMR, terminal_b.id) == TRUE:` 
+                            - if the cooling system is purchased CHW and heating fluid loop serves a boiler then SYS-1a: `if is_fluid_loop_purchased_CHW(B_RMR, hvac_b.id) == TRUE AND is_fluid_loop_attached_to_boiler(B_RMR, hvac_b.id) == TRUE: is_baseline_system_1 = "SYS-1a"`
+                            - elif the cooling system is purchased CHW and heating system is purchased heating then SYS-1c: `if is_fluid_loop_purchased_CHW(B_RMR, hvac_b.id) == TRUE AND is_fluid_loop_purchased_heating(B_RMR, hvac_b.id) == TRUE: is_baseline_system_1 = "SYS-1c"`
+                            - Else, is_baseline_system_1 = "Not_Sys_1": `is_baseline_system_1 = Not_Sys_1`  
+                    - Else, is_baseline_system_1 = "Not_Sys_1": `is_baseline_system_1 = Not_Sys_1`
+            - Else, is_baseline_system_1 = "Not_Sys_1": `is_baseline_system_1 = Not_Sys_1`
+        - Else, is_baseline_system_1 = "Not_Sys_1": `is_baseline_system_1 = Not_Sys_1`
+    - Else, is_baseline_system_1 = "Not_Sys_1": `is_baseline_system_1 = Not_Sys_1`
+- Else, is_baseline_system_1 = "Not_Sys_1": `is_baseline_system_1 = Not_Sys_1`
```

Comment:
If we start the function with is_baseline_system_1 = Not_Sys_1, then we could get rid of lines 46-51.